discord/commands.py

```diff
@@ -0,0 +1,557 @@
+from models import Variable, User, RwrAccount, RwrAccountStat
+from flask_discord_interactions.models.embed import Field
+from flask_discord_interactions import Message, Permission
+from rwrs import app, cache, db, discord_interactions
+from . import constants, utils, embeds
+from rwr.player import Player
+import steam_helpers
+import rwr.scraper
+import arrow
+import os
+
+maintenance_command_group = discord_interactions.command_group(
+    'maintenance',
+    default_permission=False,
+    permissions=[
+        Permission(user=constants.MY_DISCORD_ID)
+    ]
+)
+
+motd_command_group = discord_interactions.command_group(
+    'motd',
+    default_permission=False,
+    permissions=[
+        Permission(user=constants.MY_DISCORD_ID)
+    ]
+)
+
+user_command_group = discord_interactions.command_group(
+    'user',
+    default_permission=False,
+    permissions=[
+        Permission(user=constants.MY_DISCORD_ID)
+    ]
+)
+
+user_api_command_subgroup = user_command_group.subgroup('api')
+
+event_command_group = discord_interactions.command_group(
+    'event',
+    default_permission=False,
+    permissions=[
+        Permission(user=constants.MY_DISCORD_ID)
+    ]
+)
+
+
+@discord_interactions.command(
+    'cc',
+    'Clears RWRS cache',
+    default_permission=False,
+    permissions=[
+        Permission(user=constants.MY_DISCORD_ID)
+    ]
+)
+def cc(
+    ctx
+):
+    cache.clear()
+
+    return Message('Cache cleared.', ephemeral=True)
+
+
```

Comment:
Add command `/update-commands` which updates commands in Discord (invokes discord_interactions.update_commands)